Report Generation/Report_graphs.py

```diff
@@ -0,0 +1,75 @@
+import pandas as pd
+import psycopg2
+from io import BytesIO
+import matplotlib.pyplot as plt 
+
+class data_generation:
```

Comment:
Tbh, not a big fan of doing it this way. `data_generation` sounds more like a function than an object (and the only attribute you have of this object is `data`)